integrations/helm/chartsync/chartsync.go

```diff
@@ -385,7 +385,7 @@ func (chs *ChartChangeSync) reapplyReleaseDefs() error {
 	}
 
 	for _, fhr := range resources {
-		chs.reconcileReleaseDef(fhr)
+		chs.reconcileReleaseDef(fhr, "")
```

Comment:
This is going to (potentially) reset the revision, if it re-releases a chart. I don't think that's what you want to do, since it will arbitrarily remove information that's still relevant.

Reply:
I feared that would be the case. Changed the approach.